CHANGELOG.md

```diff
@@ -3,6 +3,15 @@
 All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog].
 
+## Unreleased
+### Bugs fixed
+* ivy doesn't convert all variables to string when sorting or calling
+  `prescient-remember`, so it need to preprocess when work with
+  `ivy-prescient.el`. if not, n `wrong-type-argument` error may 
+  occur ([#119]).
+
+[#119]: https://github.com/raxod502/prescient.el/pull/119
```

Comment:
I don't suppose you know offhand what command(s) this affects? Would help in explaining the impact to users. Not needed though, just asking.